examples/fetch_xpubs_as_admin/fetch_xpubs_as_admin.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"context"
+	"log"
+
+	wallet "github.com/bitcoin-sv/spv-wallet-go-client"
+	"github.com/bitcoin-sv/spv-wallet-go-client/examples"
+	"github.com/bitcoin-sv/spv-wallet-go-client/examples/exampleutil"
+)
+
+func main() {
+	adminAPI, err := wallet.NewAdminAPIWithXPriv(exampleutil.ExampleConfig, examples.XPriv)
```

Comment:
It would be good to have example with query options not just a simple call for all.